packages/frontend-acceptance-tests/src/features/2 - 12 Month Licences/2_junior_licence-12-months_immediate_start.feature

```diff
@@ -49,7 +49,7 @@ Feature: I want to buy a junior annual fishing
     Then I am on the licence summary page and I click continue
     And I enter "3" and "SN153PG" as my house number and postcode
     And I select "100121002711" as an address
-    And I do not have either of these
+    And I enter email as "email@gmail.com"
```

Comment:
```suggestion
    And I enter email as "email@gmail.com"
```

We should really be using example.com for all our dummy emails rather than gmail. I know we're doing this elsewhere already so might be worth creating a ticket to update that.